src/main/client/src/reducers/table/actions.js

```diff
@@ -206,6 +206,11 @@ export const searchWorkspaceFeatures = (
                 type: types.ADD_SEARCH_RESULTS_LAYER,
                 layers: addSearchResultLayers,
             });
+
+            layersToBeAdded.layers.map(l => dispatch({
```

Comment:
1) Dispatchaa n -kappaletta actioneita, aiheuttaako ongelman siinä että tulee ns. turhia renderöintejä. Voisiko esim. tehdä typen, jossa layerId:n sijalla olisi esim layerIds: string[]

2) Jos käytetään Array.map, niin silloin sitä paluuarvoa tulisi käyttää (kun se kuitenkin luo uuden arrayn, tässä tapauksessa [undefined, undefined, ...]. Eli ton vois vaihtaa käyttämään Array.forEach

Reply:
Luotu uusi action joka ottaa vastaan listan ID:itä